scripts/west_commands/build.py

```diff
@@ -456,16 +456,34 @@ def _update_cache(self):
         with contextlib.suppress(FileNotFoundError):
             self.cmake_cache = CMakeCache.from_build_dir(self.build_dir)
 
+    def _get_dir_fmt_context(self):
+        # Return a dictionary of build attributes which are used while
+        # substituting the placeholders in the build.dir-fmt format string.
+        west_top_dir = west_topdir()
```

Comment:
```suggestion
        west_topdir = west_topdir()
```

It's called `topdir` everywhere else, so why the extra `_` in `top_dir` ?
https://docs.zephyrproject.org/latest/develop/west/basics.html

Reply:
I wanted to avoid that the imported west_topdir is shadowed. Is this common?